x/nft/model.go

```diff
@@ -49,15 +49,15 @@ func (u *NonFungibleToken) OwnerAddress() weave.Address {
 }
 
 func (m *NonFungibleToken) Approvals() *ApprovalOps {
-	return NewApprovalOps(m.OwnerAddress(), &m.ActionApprovals)
```

Comment:
Good changes. Can you now remove the `OwnerAddress` method above, as everyone will just use Owner directly now?